docs/source/algorithms/PolarizerEfficiency-v1.rst

```diff
@@ -0,0 +1,54 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Calculates how the efficiency of a polarizer varies with wavelength. The
+ordering of the workspaces in ``InputWorkspace`` is taken from the ``SpinStates`` parameter, and the analyser
+efficiency, :math:`\epsilon_{cell}`, is given by ``AnalyserEfficiency``.
+
+The polarization of the polarizer, :math:`P_{SM}`, is given by
+
+.. math::
+    P_{SM} = \frac{T_{00} - T_{01}}{2P_{cell}(T_{00} + T_{01})}
+
+Since the efficiency, :math:`\epsilon_{SM}`, is given by :math:`\frac{1 + P_{SM}}{2}`, we have that
+
+.. math::
+    \epsilon_{SM} = \frac{1}{2} + \frac{T_{00} - T_{01}}{4(2\epsilon_{cell} - 1)(T_{00} + T_{01})}
+
```

Comment:
I noticed in the code that one of the workspaces is rebinned during the calculation. Could we add a note in the docs here to mention which workspace's binning is the one that's used for the output?